.github/workflows/server-integration-tests.yml

```
@@ -0,0 +1,104 @@
+        uses: actions/checkout@v4
+        with:
+          ref: "refs/pull/${{ inputs.pr }}/merge"
+
+      # Checkout the code in the current branch in case the workflow is called because of a branch push event
+      - name: Checkout the head commit of the branch
+        if: inputs.pr == 0
+        uses: actions/checkout@v4
+
+      # Setup Java
+      - name: Set up JDK 17
+        if: steps.run_result.outputs.run_result != 'success'  && (steps.changed-files-specific.outputs.any_changed == 'true' ||  github.event_name == 'push' || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule')
```

actionlint (1.7.4)

property "run_result" is not defined in object type {}

(expression)